src/pages/index.tsx

```diff
@@ -11,8 +11,12 @@ import EditorErrorBoundary from '~components/errorBoundaries/EditorErrorBoundary
 import Editor from '~components/editor/Editor'
 import { InspectorProvider } from '~contexts/inspector-context'
 import Inspector from '~components/inspector/Inspector'
+import { HiOutlineDesktopComputer } from 'react-icons/hi'
+import { MdOutlineTabletMac } from 'react-icons/md'
+import { ImMobile } from 'react-icons/im'
 
 const App = () => {
+  const [editorWidth, setEditorWidth] = useState<string>('100%')
```

Comment:
Can you move this state in the redux state app ? 
https://github.com/premieroctet/openchakra/blob/master/src/core/models/app.ts#L5